### ISSUE TITLE
Intensity range doesn't default to reasonable values when selected for fitting

### ISSUE THREAD

**Issue by mdoucet** [reflectometry/refl1d#179]
After creating a model with the builder and loading data into it, one then selects parameters to fit.
If any parameter is selected, is usually appears on the summary tab with a reasonable range around its current value. The intensity doesn't do that and the range boxes are left blank. One has to enter the min and max by hand before fitting.

**Comment by bmaranville** [comment]
Yes, this is a good idea.  What about a simple hard coded range from 0.8 to 1.2?

**Comment by andyfaff** [comment]
That's fine if it's normalized, but xrr data often isn't.

**Comment by bmaranville** [comment]
If they are not normalized, do they have resolution defined or any other reduction steps typically?  This is a good discussion for ORSO, but I feel like making widely available reduction available is preferable to fitting footprint etc

**Comment by bmaranville** [comment]
Though it wouldn't take much to add footprint and constant background to a model...

**Comment by mdoucet** [comment]
This is interesting. I'm a little biased because our data come out of the reduction normalized, so it's "supposed" to be 1. So 0.8 to 1.2 is a good default then. But this might not be true for others. Even if the default range was -inf to inf I'd be OK with that. At the moment the default scale is one, but it won't fit until you enter a range manually (which users will forget and complain about). I'd be OK with -inf to inf if that works for everyone. As long as there's a default.

**Comment by bmaranville** [comment]
Null should be converted to -inf and inf and it should let you fit.  If that's not happening it's a different bug...

**Comment by bmaranville** [comment]
The prior is indeed being set to -inf to inf when the bounds are empty (like they are after you add intensity as a fitted parameter), and the fit _nearly_ completes...

The error is triggered when the fit message is generated and it tries to format the fit range for that parameter, and it's looking for parameter.bounds.  I've fixed it so it looks for parameter.prior.bounds (which includes the limits and user bounds), then falls back to parameter.bounds, then falls back to the limits (which are defined for every parameter)

[bumps/bumps](https://github.com/bumps/bumps/commit/e91aa5d595f1bf073daaf5a4c2e19e97dd37bfbf)
